KrispyTech/airneis
# Airneis 167 - Backoffice - Add Product

# DONE 

Added the add-product page in the backoffice, connected to back

# TEST

- Go to /backoffice/products
- Click on "Add"
- Fill the form with your very own product
- Click on "Validate"
- Go check your DB: your product should now be in it

# VISUAL
![Screenshot_2024-06-28_11-16-10](https://github.com/KrispyTech/airneis/assets/137942699/95c68657-ae52-4f87-812a-e7993333e63b)
